muskathlon/models/survey_user_input.py

```diff
@@ -31,12 +31,19 @@ def write(self, vals):
                     'muskathlon.stage_fill_profile').id)
             ])
 
+            survey = registrations.event_id.medical_survey_id
```

Comment:
Be careful because `registrations` could potentially be a recordset (not a single record)
```suggestion
            surveys = registrations.mapped('event_id.medical_survey_id')
```